tests/naive.py

```diff
@@ -1757,3 +1788,47 @@ def _total_diagonal_ndists(tile_lower_diag, tile_upper_diag, tile_height, tile_w
         )
 
     return total_ndists
+
+
+def merge_topk_PI(PA, PB, IA, IB):
+    profile = np.column_stack((PA, PB))
+    indices = np.column_stack((IA, IB))
+
+    idx = np.argsort(profile, axis=1)
+    profile[:, :] = np.take_along_axis(profile, idx, axis=1)
+    indices[:, :] = np.take_along_axis(indices, idx, axis=1)
+
+    PA[:, :] = profile[:, : PA.shape[1]]
+    IA[:, :] = indices[:, : PA.shape[1]]
+
+
+def merge_topk_ρI(ρA, ρB, IA, IB):
```

Comment:
Improve the docstring by:
* making it shorted
* making it more precise
* providing a better example

Any suggestion?

Reply:
I care less here as this is the naive version